inventory/sample.gcp.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -35,6 +35,32 @@ openshift_enable_service_catalog: false
 openshift_hosted_prometheus_deploy: false
 openshift_cfme_install_app: false
 
+openshift_node_groups:
+  - name: node-config-master 
+    labels:
+      - 'node-role.kubernetes.io/master=true' 
```

Comment:
Also, to be super-picky - some of this yaml is off on the indentation. I believe it works as-is, but for readability please check these lines (41, 49 & 57) compared to lines 43-46, 51-54 and 59-62.